src/sentry/static/sentry/app/api.tsx

```diff
@@ -299,13 +299,25 @@ export class Client {
    */
   request(path: string, options: Readonly<RequestOptions> = {}): Request {
     const method = options.method || (options.data ? 'POST' : 'GET');
+
+    let fullUrl = buildRequestUrl(this.baseUrl, path, options.query);
+
     let data = options.data;
 
     if (!isUndefined(data) && method !== 'GET') {
       data = JSON.stringify(data);
     }
 
-    const fullUrl = buildRequestUrl(this.baseUrl, path, options.query);
+    // TODO(epurkhiser): Mimicking the old jQuery API, data could be a string /
+    // object for GET requets. jQuery just sticks it onto the URL as query
+    // parameters
+    if (method === 'GET' && data) {
```

Comment:
I would like to track down where we're passing in `data` for `GET` requests later. But for now this just keeps the interface the same